src/form/v2/death/forms/pages/eventDetails.ts

```diff
@@ -0,0 +1,325 @@
+  MEDICALLY_CERTIFIED: {
+    defaultMessage: 'Medically Certified Cause of Death',
+    description: 'Option for form field: Method of Cause of Death',
+    id: 'form.field.label.medicallyCertified'
+  }
+} satisfies Record<keyof typeof SourceCauseDeathType, TranslationConfig>
+
+const sourceCauseDeathOptions = createSelectOptions(
+  SourceCauseDeathType,
+  sourceCauseDeathMessageDescriptors
+)
+
+export const PlaceOfDeath = {
+  HEALTH_FACILITY: 'HEALTH_FACILITY',
+  DECEASED_USUAL_RESIDENCE: 'DECEASED_USUAL_RESIDENCE',
+  OTHER: 'OTHER'
+} as const
+
+const placeOfDeathMessageDescriptors = {
+  HEALTH_FACILITY: {
+    defaultMessage: 'Health Institution',
+    description: 'Select item for Health Institution',
+    id: 'v2.form.field.label.healthInstitution'
+  },
+  DECEASED_USUAL_RESIDENCE: {
+    defaultMessage: "Deceased's usual place of residence",
+    description:
+      'Option for place of occurrence of death same as deceased primary address',
+    id: 'v2.form.field.label.placeOfDeathSameAsPrimary'
+  },
+  OTHER: {
+    defaultMessage: 'Other',
+    description: 'Select item for Other location',
+    id: 'v2.form.field.label.otherInstitution'
+  }
+} satisfies Record<keyof typeof PlaceOfDeath, TranslationConfig>
+
+const placeOfDeathOptions = createSelectOptions(
+  PlaceOfDeath,
+  placeOfDeathMessageDescriptors
+)
+
+export const eventDetails = defineFormPage({
+  id: 'eventDetails',
+  type: PageTypes.enum.FORM,
+  title: {
+    defaultMessage: 'Event details',
+    description: 'Form section title for event details',
+    id: 'v2.form.death.eventDetails.title'
+  },
+  fields: [
+    {
+      id: 'event.date',
+      type: 'DATE',
+      required: true,
+      validation: [
+        {
+          message: {
+            defaultMessage: 'Must be a valid date',
+            description: 'This is the error message for invalid date',
+            id: 'v2.event.death.action.declare.form.section.event.field.date.error'
+          },
+          validator: field('event.date').isBefore().now()
+        }
+      ],
+      label: {
+        defaultMessage: 'Date of death',
+        description: 'This is the label for the field',
+        id: 'v2.event.death.action.declare.form.section.event.field.date.label'
+      }
+    },
+    {
+      id: 'event.reasonForLateRegistration',
+      type: FieldType.TEXT,
+      required: true,
+      label: {
+        defaultMessage: 'Reason',
+        description: 'This is the label for the field',
+        id: 'v2.event.death.action.declare.form.section.event.field.reason.label'
+      },
+      conditionals: [
+        {
+          type: ConditionalType.SHOW,
+          conditional: and(
+            not(
+              field('event.date')
+                .isAfter()
+                .days(applicationConfig.DEATH.REGISTRATION_TARGET)
+                .inPast()
+            ),
+            field('event.date').isBefore().now()
+          )
+        }
+      ]
+    },
+    {
+      id: 'event.manner',
```

Comment:
```suggestion
      id: 'event.mannerOfDeath',
```
I think more explicit is better.